gradle.properties

```diff
@@ -16,3 +16,4 @@ android.useAndroidX=true
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 org.gradle.jvmargs=-Xmx8192m
+android.nonTransitiveRClass=false
```

Comment:
Nope, remove this, this is true by default now

Reply:
My bad, I thought I should use `Simple-Thank-You` as reference and it has it set to false.